cmake/version_config.hpp.in

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+#define RAPIDSAI_RMM_VERSION_MAJOR @RMM_VERSION_MAJOR@
```

Comment:
I would like to discuss the preferred naming convention for this with the RAPIDS team before merging this.

Reply:
Got it. The XGBoost project will block its release until this PR is resolved.